src/modules/imageresizer/tests/Properties/SettingsTests.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace ImageResizer.Properties
 {
-    public class SettingsTests
+    public class SettingsTests : IClassFixture<AppFixture>
```

Comment:
Settings.Reload() requires the App to be created as it needs to change the Settings properties on the [UI thread using App.Current.Dispatcher.Invoke()](https://github.com/arjunbalgovind/PowerToys/blob/5f0c08b960152ee3aa42a2f7b3a7f2750b537aaf/src/modules/imageresizer/ui/Properties/Settings.cs#L411). If the properties are directly assigned it throws an error since those properties are bound to the UI.
The App variable needs to be created before Settings.Reload() is called as App.Current would be a null reference otherwise. The App variable can be created only once on the SettingsTests since multiple calls to it causes `System.InvalidOperationException : Cannot create more than one System.Windows.Application instance in the same AppDomain`.
Thanks for calling that out, I thought I had added comments to explain that behavior but it seems it got removed in an earlier commit.